spec/helpers/application_helper_spec.rb

```diff
@@ -227,14 +227,27 @@ def current_user_can_publish? = false
       )
     end
 
-    it "does not return an 'last published' row when the manual does not have publish tasks" do
+    it "does not return a 'last published' row when the manual does not have publish tasks" do
       manual = FactoryBot.build_stubbed(:manual)
       manual.publish_tasks = []
 
       rows = manual_metadata_rows(manual)
 
       expect(rows).not_to include(include(key: "Last published"))
     end
+
+    it "indicates when originally published time will be used as public timestamp" do
+      manual = FactoryBot.build_stubbed(:manual, use_originally_published_at_for_public_timestamp: true)
+      allow_any_instance_of(ApplicationHelper).to receive(:publication_task_state).and_return("test publication task state")
```

Comment:
If we're going to mock this (would it be possible not to using timecop or something?) do we want to check that it received the first item from `publish_tasks` as a parameter when `publication_task_state` was called?

Reply:
Oh I was just wondering if you'd mocked the `publication_task_state` helper method because it was time sensitive, in which case we could use the real function and use Timecop to freeze time so we could make a predictable assertion without mocking the method. But that's probably not the case. If that doesn't clarify things, ping me and we can jump on a call. It's not particularly important anyway

Reply:
Ah, no, it's more to avoid having to worry about the logic of what that method returns, to keep this test more "unity".